### ISSUE TITLE
Descent II installer using WineVDM stops in DirectX setup

### ISSUE THREAD

**Issue by BEENNath58** [otya128/winevdm#1163]
Descent II for Windows uses a InstallShield setup program to install the game. When installing the game on an unsupported 32-bit system (such as Windows XP), the installation goes smoothly, but on Windows 11 with WineVDM, it stops at a DirectX check.

The DirectX check should not stop the installation, it should go through and install the game.

This is the place where I encounter the problem.
![descent2setupstop](https://user-images.githubusercontent.com/64649734/170941017-15bc98d6-a2e7-4bc8-892e-4b551e8243ed.png)

I am running on Windows 11 21H2.

I am not able to provide a trace.txt file because the game's InstallShield needs to have the Win95VersionLie shim which I can apply only when the game is launched using the Autorun.exe program (shims get shared in son processes).  I am not aware of a method to apply shims on IS files. Thus if I try to trace the autorun.exe file until the InstallShield DirectX installation problem, trace.txt is still empty.

**Comment by cracyc** [comment]
That error is likely from the win32 part of setup.  I doubt there's anything that can be done to winevdm to fix that.

**Comment by BEENNath58** [comment]
> That error is likely from the win32 part of setup. I doubt there's anything that can be done to winevdm to fix that.

I don't think that it's the case. I ran the same application just now on Windows 7 Service Pack 1 VM with WINEVDM and there was no such error. The installation completed there successfully.
